proto/buf.yaml

```diff
@@ -2,7 +2,7 @@ version: v1
 name: buf.build/cosmwasm/wasmd
 deps:
   - buf.build/cosmos/cosmos-proto
-  - buf.build/cosmos/cosmos-sdk:2aa7ff2b23df473a85b7a7fe1884105d #v0.47.0-rc
+  - buf.build/cosmos/cosmos-sdk:8c515ebc07ee4514aabcaf3a584a1d1a #v0.47.0
```

Comment:
```suggestion
  - buf.build/cosmos/cosmos-sdk:v0.47.0
```

Reply:
This is nice! Thanks for letting me know